spark/src/test/scala/org/apache/spark/sql/delta/DeleteSQLSuite.scala

```diff
@@ -82,6 +82,26 @@ class DeleteSQLSuite extends DeleteSuiteBase
       }
     }
   }
+
+  test("DELETE with huge predicate") {
```

Comment:
can we have 3 tests
* making it shorter works (we have that)
* setting nothing truncates
* the limit can be removed / we can actually still store very long predicates